Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_suspiciousdomain",
+    "version": -1,
+    "modified": "2023-11-09T22:08:56.499540505+05:30",
+    "name": "Suspicious domain",
+    "ownerOnly": false,
+    "description": "Suspicious Domain",
+    "cliName": "suspiciousdomain",
+    "type": "shortText",
```

Comment:
@syed-loginsoft - I thought we discussed the _Suspicious domain_ will be a list.

Reply:
@kgal-pan Should @syed-loginsoft change the type to 'list'?

Reply:
https://github.com/demisto/content/pull/30787#discussion_r1398367581